js/stdlib/ts/ETH_like.ts

```diff
@@ -74,7 +74,7 @@ export type {
 import type { // =>
   Stdlib_Backend
 } from './interfaces';
-import { setQueryLowerBound, getQueryLowerBound } from './shared_impl';
+import { setQueryLowerBound, getQueryLowerBound, formatWithDecimals } from './shared_impl';
```

Comment:
Can you consider whether we should use this function to replace the one 1066 and unify the implementations of that too (basically copying `formatCurrency` from `ALGO.ts` with a different default)